src/officehours_ui/static/css/main.css

```diff
@@ -687,3 +687,16 @@ div[role=main] {
 .badge-success {
 	background-color: var(--custom-green);
 }
+
+a.nav-link:hover {
+	background-color: #004881;
+	border-radius: 3px;
+}
+
+a.dropdown-item:hover {
+	background-color: #C0C0C0 !important;
+}
+
+input.dropdown-item:hover {
+	background-color: #C0C0C0 !important;
+}
```

Comment:
@vjcao, are both the `a` and `input` blocks necessary for this? I tried commenting out the `input` block, nothing seems to have changed?

Reply:
Hi @ssciolla. Both of the blocks are necessary for the dropdown menu. The `a` block is for the "Manage Queues" and "Preferences" buttons while the `input` block is for the "Logout" button. Sometimes changes to static files won't be reflected on local host so I sometimes have to use this command to refresh the static files: `python manage.py collectstatic --noinput --clear`.

Reply:
Ah, you are correct. Logout is not highlighting on hover. I'll resolve this comment.